### PR TITLE
Release/1.0.1

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
-Release notes added
-Subtitle changed

### CODE REVIEW

File: widget/release_notes.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # What's New
-Minor bug fix:
+Minor bug fixes.
+ - Widget now remembers a change in 'Picklist' value.
```
Comment (cs-suyog-jinturkar) [inline widget/release_notes.md]: Minor bug fixes:

Comment (cs-suyog-jinturkar) [reply widget/release_notes.md]: Remembers?

File: widget/info.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "fieldsOfInterest",
     "title": "Fields Of Interest",
-    "subTitle": "Fields Of Interest",
+    "subTitle": "Select fields to display in the detailed view, regardless of the 'Visiblity constraint'",
```
Comment (cs-suyog-jinturkar) [inline widget/info.json:4]: This needs to be reworked. Please get one string from Abhishek

File: widget/release_notes.md (hunk 1 above)
Comment (cs-yash-patodkar) [reply widget/release_notes.md]: 'Updates'?

Comment (cs-suyog-jinturkar) [reply widget/release_notes.md]: Check with namarata, what is a better word here.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**